content/en/blog/_posts/2017-11-00-Containerd-Container-Runtime-Options-Kubernetes.md

```diff
@@ -56,13 +56,13 @@ Cri-containerd uses containerd to manage the full container lifecycle and all co
 
 Let’s use an example to demonstrate how cri-containerd works for the case when Kubelet creates a single-container pod:  
 
-1. 1.Kubelet calls cri-containerd, via the CRI runtime service API, to create a pod;
-2. 2.cri-containerd uses containerd to create and start a special [pause container](https://www.ianlewis.org/en/almighty-pause-container) (the _sandbox container_) and put that container inside the pod’s cgroups and namespace (steps omitted for brevity);
-3. 3.cri-containerd configures the pod’s network namespace using CNI;
-4. 4.Kubelet subsequently calls cri-containerd, via the CRI image service API, to pull the application container image;
-5. 5.cri-containerd further uses containerd to pull the image if the image is not present on the node;
-6. 6.Kubelet then calls cri-containerd, via the CRI runtime service API, to create and start the application container inside the pod using the pulled container image;
-7. 7.cri-containerd finally calls containerd to create the application container, put it inside the pod’s cgroups and namespace, then to start the pod’s new application container.
+1. Kubelet calls cri-containerd, via the CRI runtime service API, to create a pod;
+2. cri-containerd uses containerd to create and start a special [pause container](https://www.ianlewis.org/en/almighty-pause-container) (the _sandbox container_) and put that container inside the pod’s cgroups and namespace (steps omitted for brevity);
+3. cri-containerd configures the pod’s network namespace using CNI;
+4. Kubelet subsequently calls cri-containerd, via the CRI image service API, to pull the application container image;
+5. cri-containerd further uses containerd to pull the image if the image is not present on the node;
+6. Kubelet then calls cri-containerd, via the CRI runtime service API, to create and start the application using the pulled container image;
+7. cri-containerd finally calls containerd to put the application container inside the pod’s cgroups and namespace.
 After these steps, a pod and its corresponding application container is created and running.
```

Comment:
I'm not sure if the intention is to have this line be a part of the last step, if not, add a carriage return before this line to keep the markdown renderer from adding it to step 7.

Reply:
Hello @nakamume .
I have reservations about this change. I am not sure whether step 7 needs to be changed.
I think the author is introducing step 7 by leading with the text from step 6?

- Does `cri-containerd` use or delegate to `containerd` to create the application container? OR
- Does `cri-containerd` create the app. container and `containerd` puts the container newly create container in the cgroups and namespaces? The difference is subtle.

Reply:
> I have reservations about this change. I am not sure whether step 7 needs to be changed.
> 
> I think the author is introducing step 7 by leading with the text from step 6?

I think I agree with you here (and since in my original comment I say I'm not sure of the intention, I've removed the suggested change to be consistent).

Reply:
Thanks Guys. Removed the changes to text here https://github.com/kubernetes/website/pull/24718/commits/fd1c77c01acc249254dbcdc07f77f3c6eae24c5b

Reply:
@nakamume it looks like the last commit has removed the "After these steps..." line.

Reply:
Hi @nakamume just following up here. Please add back the "After these steps.." line, thank you!